src/runtime/utils/helpers.ts

```diff
@@ -159,17 +159,58 @@ export async function addNewStaticType(sectionTypeName: string): Promise<{ statu
   }
 }
 
+export const abstractPathLanguage = (finalPath: any) => {
```

Comment:
add a comment to this function
If this is an extract from i18n internal functions explain in your comment why you couldn't just call it
Otherwise, explain in your comment why you couldn't use the i18n function and why you had to rewrite a function here